src/renderer/screens/Preferences/Basic.js

```diff
@@ -55,6 +62,10 @@ function BasicPreferences(props) {
     }
   };
 
+  useEffectOnce(() => {
```

Comment:
We shouldn't need useEffectOnce here.  uEO is a "weird" thing that's kind of not the React Way.

why doesn't regular useEffect work?

Reply:
I suppose regular useEffect would work here too.